benchmark/benchmark_runner.cpp

```diff
@@ -181,6 +183,25 @@ void LoadInterpretedBenchmarks() {
 	});
 }
 
+string parse_root_dir_or_default(const int arg_counter, char const *const *arg_values) {
+	// default root directory is the benchmark directory in the duckdb root directory
+	string root_dir = DUCKDB_ROOT_DIRECTORY;
+
+	// check if the user specified a different root directory
+	for (int arg_index = 1; arg_index < arg_counter; ++arg_index) {
+		string arg = arg_values[arg_index];
+		if (arg == "--root-dir") {
+			if (arg_index + 1 >= arg_counter) {
+				fprintf(stderr, "Missing argument for --root-dir\n");
+				print_help();
+				exit(1);
+			}
+			root_dir = FileSystem::JoinPath(FileSystem::GetWorkingDirectory(), arg_values[arg_index + 1]);
```

Comment:
Does this always need to be a relative path?

Reply:
Good catch! think I fixed it now.